src/kruskal_wallis.jl

```diff
@@ -32,6 +32,35 @@ immutable KruskalWallisTest <: HypothesisTest
     tie_adjustment::Float64  # adjustment for ties
 end
 
+"""
+    KruskalWallisTest{T<:Real}(groups::AbstractVector{T}...)
+
+Perform Kruskal Wallis rank sum test of the null hypothesis that the location parameters of
+the distribution of the ``n`` observations are the same in each of the `groups`
+``\\mathcal{G}`` against the alternative hypothesis that they differ in at least one.
```

Comment:
"at least one" sounds weird to me. I can't find this description in other references.